docker_base/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# This is a base container we use. It has pgsql, redis and Python 3.5,
+# plus a user named qabel and locales.
+
+FROM ubuntu:16.04
```

Comment:
Ok sounds reasonable. As the "normal" use would be to just link a postgres container nobody complies about alpine linux not support it ^^